packages/validation/src/no.ts

```diff
@@ -159,3 +159,9 @@ export function validateNationalIdentityNumber(
 
   return isValidDate(year, month, day);
 }
+
+export function validateAccountNumber(value: string): boolean {
```

Comment:
Kan vi få inn noe jsdoc her? Se de andre metodene for eksempler/inspirasjon.

Reply:
ahh ja glemte det!